config/cookieconsent.php

```diff
@@ -55,6 +55,19 @@
 
     'policy' => null,
 
+    /*
+    |--------------------------------------------------------------------------
+    | CSP configuration
+    |--------------------------------------------------------------------------
+    |
+    | Most cookie notices display a link to a dedicated page explaining
+    | the extended cookies usage policy. If your application has such a page
+    | you can add its route name here.
+    |
+    */
```

Comment:
Please add a specific description for the CSP configuration, including:
- What it relates to (you may add a link to Spatie's CSP package)
- How to configure it for Spatie's CSP package: `env('CSP_ENABLE', false) && env('CSP_NONCE_ENABLED', true)`